operator/v1/types_console.go

```diff
@@ -143,8 +143,133 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeMode is the value of the logo theme mode that determines the theme mode in the console UI.
+// +kubebuilder:validation:Enum="Dark";"Light"
+// +enum
+type ThemeMode string
+
+// ThemeMode values
+const (
+	// ThemeModeDark represents the dark theme for the console UI.
+	ThemeModeDark ThemeMode = "Dark"
+
+	// ThemeModeLight represents the light theme for the console UI.
+	ThemeModeLight ThemeMode = "Light"
```

Comment:
Nit for terminology consistency:
```suggestion
	// ThemeModeDark represents the dark mode for a console theme.
	ThemeModeDark ThemeMode = "Dark"

	// ThemeModeLight represents the light mode for a console theme.
	ThemeModeLight ThemeMode = "Light"
```